.travis.yml

```diff
@@ -13,7 +13,7 @@ script:
   # https://github.com/shipwright-io/build/issues/123
   - make test-unit-coverage
   - make test-integration
-  - make test-e2e TEST_IMAGE_REPO="$(./hack/install-registry.sh show):5000/shipwright-io/build-e2e"
+  - travis_wait 30 make test-e2e TEST_IMAGE_REPO="$(./hack/install-registry.sh show):5000/shipwright-io/build-e2e"
```

Comment:
@SaschaSchwarze0 can you try dropping this? Having access to the log stream in Travis is a nice feature.